docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,184 @@
+---
+sidebar_label: "Using DataVolumes for disk templates"
+title: "Using DataVolumes for disk templates"
+description: "Using DataVolumes to quickly create VMs from template disks in Palette Virtual Machine Orchestrator"
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vmo"]
+---
+
+
+
+When VMs are deployed from template, they almost always include a disk template containing a preinstalled copy of the desired OS. The Kubevirt CDI is used to import these templates through a concept called [Data Volumes](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/datavolumes.md) and it supports multiple different approaches to do so. Some approaches are vastly quicker than others though. Hence, designing this correctly is important. This page details how to do that.
+
+
+## Overview
+
+![Image cloning with CDI](/cdi-disk-clone.png)
+
+The diagram above is a representation of best approach:
+1. Create a `DataVolume` resource to import an external disk image into the cluster, saving it as a template PVC on the cluster
+2. Configure a `vmTemplate` resource to clone the template PVC into a new PVC for use by the VM. This is also achieved through a `DataVolume` specification, but this time inside the VM specification.
+3. Set up RBAC permissions within the cluster to ensure Data Volumes can be used across namespaces.
+4. Define `StorageProfile` resources to ensure Kubevirt CDI uses the CSI-assisted cloning technique instead of any slower alternatives.
+
+<br />
+
+## Implementation
+
+The following reference implementation details the recommended approach to implementing the components mentioned above. Version 4.2.0 and higher of the VMO pack contain native support for this approach:
+* Template disks will be imported into the `vmo-golden-images` namespace
+* The `default` and `virtual-machines` namespaces will be configured with RBAC permissions required for cloning DataVolumes from the `vmo-golden-images` namespace
+* `vmTemplate` resources will use a `dataVolumeTemplates` section that clones the template disk from the `vmo-golden-images` namespace
+
+<br />
+
+### Importing disk images as templates
+
+The following manifest can be added to a manifest layer in a Palette cluster profile to define a `DataVolume` that imports the example disk template for Ubuntu 22.04 into the `vmo-golden-images` namespace:
```

Comment:
Can you provide the cluster profile to add this manifest to?  This will make it easier for the learner to follow along and reproduce the steps in your tutorial.